src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo/PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  posts: Post[],
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => {
+  return (
+    <>
+      {posts.map(post => (
+        <div className="PostList">
+          <PostInfo key={post.id} post={post} />
```

Comment:
```suggestion
    <div className="PostList">
      {posts.map(post => (
        <PostInfo key={post.id} post={post} />
```

Reply:
not fixed 
![image](https://github.com/mate-academy/react_static-list-of-posts/assets/122989585/84e8eebd-541a-4ee8-9469-1b07dd966881)